src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
 public class Duke {
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        showLine();
+        System.out.println("\tHello! I'm Duke" + "\n\t" + "What can I do for you?");
+        showLine();
+        boolean toExit = false;
+        Scanner scan = new Scanner(System.in);
+        Task task;
+        List<Task> list = new ArrayList<>();
```

Comment:
Perhaps distinguish clearly between single-valued and multi-valued variables here?

Reply:
yes have updated in my latest version thanks